docker/zu/src/main/java/io/pravega/zookeeper/Main.kt

```diff
@@ -50,6 +50,7 @@ fun runSync(args: Array<String>, suppressOutput: Boolean = false): String {
       if (! suppressOutput) {
           print(clusterSize)
       }
+      zk.close()
```

Comment:
This is needed, otherwise the connection to the zookeeper is not closed and the program doesn't finish.
```
server.1=pipeline-zookeeper-0.pipeline-zookeeper-headless.ns-team-aep-pipeline-kafka-1-dev.svc.cluster.local:2888:3888:participant;0.0.0.0:2181
server.2=pipeline-zookeeper-1.pipeline-zookeeper-headless.ns-team-aep-pipeline-kafka-1-dev.svc.cluster.local:2888:3888:participant;0.0.0.0:2181
server.3=pipeline-zookeeper-2.pipeline-zookeeper-headless.ns-team-aep-pipeline-kafka-1-dev.svc.cluster.local:2888:3888:participant;0.0.0.0:2181
server.100=localhost:2888:3888:observer;0.0.0.0:2181
server.101=localhost:2888:3888:observer;0.0.0.0:2181
11:54:17.592 [nioEventLoopGroup-2-1] DEBUG org.apache.zookeeper.ClientCnxn - Got ping response for session id: 0x100680143a21d05 after 3ms.

11:54:18.925 [nioEventLoopGroup-2-1] DEBUG org.apache.zookeeper.ClientCnxn - Got ping response for session id: 0x100680143a21d05 after 2ms.
11:54:20.258 [nioEventLoopGroup-2-1] DEBUG org.apache.zookeeper.ClientCnxn - Got ping response for session id: 0x100680143a21d05 after 2ms.
11:54:21.591 [nioEventLoopGroup-2-1] DEBUG org.apache.zookeeper.ClientCnxn - Got ping response for session id: 0x100680143a21d05 after 2ms.
11:54:22.924 [nioEventLoopGroup-2-1] DEBUG org.apache.zookeeper.ClientCnxn - Got ping response for session id: 0x100680143a21d05 after 2ms.
11:54:24.256 [nioEventLoopGroup-2-1] DEBUG org.apache.zookeeper.ClientCnxn - Got ping response for session id: 0x100680143a21d05 after 2ms.
```